pageComponents/NewPostPage/PlayerHandSelectors.tsx

```diff
@@ -0,0 +1,117 @@
+import * as React from "react";
+import styled from "styled-components";
+import SelectablePlayingCard from "../../components/SelectablePlayingCard";
+import { MOBILE_MEDIA } from "../../constants/mediaquery";
+import PlayingCard, { Rank, Suit } from "../../models/PlayingCard";
+import getPositionByPlayerAndIndex from "../../utilities/getPositionByPlayerAndIndex";
+
+interface Props extends React.Attributes {
+  players: {
+    index: number;
+    hand: [PlayingCard | null, PlayingCard | null];
+  }[];
+  onChange?: (index: number, hand: Hand, rank: Rank, suit: Suit) => void;
```

Comment:
I think we should use prop names and signatures `players` and `onChange`, as like `value` and `onChange` of standard `<input>`. And let's consider `value` is actually `value` or should be `defaultValue`.